netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -1090,6 +1100,10 @@ net_ebpf_extension_sock_addr_authorize_recv_accept_classify(
     _net_ebpf_extension_sock_addr_copy_wfp_connection_fields(
         incoming_fixed_values, incoming_metadata_values, &net_ebpf_sock_addr_ctx);
 
+    // eBPF programs will not be invoked on connection re-auth.
```

Comment:
```suggestion
    // eBPF programs will not be invoked on connection re-authorization.
```
Avoid abbreviations, especially ambiguous ones (auth = authentication vs authorization)